cookiecutter.json

```diff
@@ -1,5 +1,6 @@
 {
     "project_name": "My New Project",
     "project_dir": "{{ cookiecutter.project_name.lower().replace(' ', '-') }}",
-    "project_source_code_dir": "src"
+    "gcp_project_name": "",
```

Comment:
not sure if this should be in the stub. Not every single project is a gcp one, right?

Reply:
Yup, not every project is deployed to GCP, but we want (?) to store Docker images in GCP repo for every new project.